### ISSUE TITLE
Changes in HTML template for latest entities

### ISSUE THREAD

**Issue by aj4m2a** [bedrock-viz/bedrock-viz#57]
The HTML template needs updating to reflect newer entities:

Heres the file I have used.

[bedrock_viz.html.template.zip](https://github.com/bedrock-viz/bedrock-viz/files/4944713/bedrock_viz.html.template.zip)

```
@@ -99,10 +99,12 @@
            <span class="caret"></span></button>
          <ul class="dropdown-menu">
	    <li><a href="#" class="entityToggle" data-type="P" data-id="19">Bat</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="122">Cat</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="122">Bee</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="75">Cat</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="10">Chicken</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="11">Cow</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="24">Donkey</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="121">Fox</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="23">Horse</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="26">Horse, Skeleton</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="27">Horse, Zombie</a></li>
@ -111,14 +113,17 @@
	    <li><a href="#" class="entityToggle" data-type="P" data-id="16">Mooshroom</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="25">Mule</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="22">Ocelot</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="123">Panda</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="113">Panda</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="30">Parrot</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="12">Pig</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="28">Polar Bear</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="18">Rabbit</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="13">Sheep</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="21">Snow Golem</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="125">Strider</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="15">Villager</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="115">Villager2</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="118">Wandering Trader</a></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="14">Wolf</a></li>
	    <li role="separator" class="divider"></li>
	    <li><a href="#" class="entityToggle" data-type="P" data-id="112">Cod</a></li>
@ -150,6 +155,9 @@
	    <li><a href="#" class="entityToggle" data-type="H" data-id="50">Guardian, Elder</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="47">Husk</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="51">NPC</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="58">Phantom</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="114">Pillager</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="59">Ravager</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="39">Silverfish</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="34">Skeleton</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="48">Skeleton, Wither</a></li>
@ -157,15 +165,20 @@
	    <li><a href="#" class="entityToggle" data-type="H" data-id="35">Spider</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="46">Stray</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="105">Vex</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="57">Vindicator</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="45">Witch</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="52">Wither</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="32">Zombie</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="44">Zombie Villager</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="116">Zombie Villager2</a></li>
	    <li role="separator" class="divider"></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="43">Blaze</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="41">Ghast</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="42">Magma Cube</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="36">Zombie Pigman</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="123">Piglin</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="124">Hoglin</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="126">Zoglin</a></li>
	    <li role="separator" class="divider"></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="53">Ender Dragon</a></li>
	    <li><a href="#" class="entityToggle" data-type="H" data-id="55">Endermite</a></li>

```

**Comment by aj4m2a** [comment]
I have the code changes ready if you assign it to me.

**Comment by cabbey** [comment]
@aj4m2a did you have a branch with these changes somewhere? I don't see a PR or a branch in your fork.

**Comment by cabbey** [comment]
I believe these are all fixed in v0.1.3
